orttraining/orttraining/python/training/orttrainer_options.py

```diff
@@ -323,18 +386,23 @@ def __init__(self, options={}):
 
         # Convert dict in object
         for k, v in self._validated_opts.items():
-            setattr(self, k, self._wrap(v))
+            setattr(self, k, self._wrap(k, v))
 
     def __repr__(self):
         return '{%s}' % str(', '.join("'%s': %s" % (k, repr(v))
                                       for (k, v) in self.__dict__.items()
                                       if k not in ['_original_opts', '_validated_opts', '_main_class_name']))
 
-    def _wrap(self, v):
+    def _wrap(self, k, v):
+        not_wrapped_fields = ['sliced_schema', 'sliced_axes']
```

Comment:
This is too hacky. We shouldn't have to blacklist the schema like this

Reply:
It's not more hacky than _ORTTrainerOptionsInternal., which assumes all dictionary should be translated to C++ struct (Python dictionary should be C++ map, not C++ struct). We need another task to redesign this part, but that is out of this PR's scope.

---
In reply to: [544667093](https://github.com/microsoft/onnxruntime/pull/5815#discussion_r544667093) [](ancestors = 544667093)

Reply:
We finalize the design offline. Thanks!

---
In reply to: [544672791](https://github.com/microsoft/onnxruntime/pull/5815#discussion_r544672791) [](ancestors = 544672791,544667093)